extensions/wikia/Email/Controller/UserRightsChangedController.class.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\Check;
+use \Email\EmailController;
+
+class UserRightsChangedController extends EmailController {
+
+	private $details;
+
+	public function initEmail() {
+		$this->details = $this->getVal( 'summary' );
```

Comment:
The call to this controller comes from everybody's favorite EmailNotification.php. "summary" just so happens to be the key for the value that we want here. Changing it [there](https://github.com/Wikia/app/blob/dev/includes/EmailNotification.php#L506-L517) would break everything else that goes through there.

I'm not sure why it's called summary and then the var in the template is called details. It looks like we have the same confusing language in all the watched page controllers.